packages/cardano-services/src/Projection/prepareTypeormProjection.ts

```diff
@@ -146,7 +148,7 @@ const storeEntities: Partial<Record<StoreName, EntityName[]>> = {
   storePoolMetricsUpdateJob: ['stakePool', 'currentPoolMetrics', 'poolMetadata'],
   storeStakeKeyRegistrations: ['block', 'stakeKeyRegistration'],
   storeStakePoolMetadataJob: ['stakePool', 'currentPoolMetrics', 'poolMetadata'],
-  storeStakePools: ['stakePool', 'currentPoolMetrics', 'poolMetadata'],
+  storeStakePools: ['stakePool', 'currentPoolMetrics', 'poolMetadata', 'poolDelisted'],
```

Comment:
This will work fine, but it's not semantically correct: `storeStakePools` can work fine without `poolDelisted` entity. There's no other way to set it up right now, so this is fine, but maybe let's leave a comment.

```suggestion
  // 'stake-pool' projection requires it, but `storeStakePools` does not.
  // at the time of writing there was no way to specify a direct projection->entity dependency.
  storeStakePools: ['stakePool', 'currentPoolMetrics', 'poolMetadata', 'poolDelisted'],
```